compiler/src/dmd/escape.d

```diff
@@ -2586,3 +2587,33 @@ private bool checkScopeVarAddr(VarDeclaration v, Expression e, Scope* sc, bool g
     return sc.setUnsafeDIP1000(gag, e.loc,
         "cannot take address of `scope` variable `%s` since `scope` applies to first indirection only", v);
 }
+
+/****************************
+ * Determine if `v` is a typesafe variadic parameter.
+ * Params:
+ *      fd = function that `v` is in
+ *      v = variable to check
+ * Returns:
+ *      true if `v` is a variadic parameter
+ */
+bool isTypesafeVariadicParameter(FuncDeclaration fd, VarDeclaration v)
```

Comment:
I think this can be refactored to avoid writing code for the parameter finding logic again, but for this PR it's fine.